### ISSUE TITLE
close 'x' button sometimes disappears

### ISSUE THREAD

**Issue by corebots** [chpii/Headerbar#25]
Hi, not sure if you still maintain this but if yes then I'd like to report this; I'm on F23 and have noticed that the 'x' button sometimes disappears; any idea how to fix this ? screenshot: http://i.imgur.com/fK6a8an.png

**Comment by DoctorJellyface** [comment]
yeah, noticed this too, just un/maximize to fix it.

**Comment by d9h02f** [comment]
Happens after viewing something in fullscreen.

**Comment by corebots** [comment]
yes i can confirm both above comments; it happens after exiting fullscreen (both a video stream and firefox window itself (hitting F11), and it gets restored after un/maximizing. still annoying though :/

**Comment by dyskette** [comment]
This is an htitle bug -> gnome-integration-team/firefox-gnome#370
